src/components/organisms/Dashboard/hooks.ts

```diff
@@ -120,8 +132,27 @@ export default (teamId?: string) => {
           : undefined,
       )
       .filter((project): project is Project => !!project);
-  }, [projectData?.projects.nodes]);
+  }, [projectNodes]);
+
+  const hasMoreProjects =
+    projectData?.projects.pageInfo?.hasNextPage || projectData?.projects.pageInfo?.hasPreviousPage;
+
+  const isRefetchingProjects = networkStatus === 3;
 
+  const getMoreProjects = useCallback(() => {
```

Comment:
The assets code needs to be updated so apologies for that, but try to remember locally declared functions should be in the `handleXXXX` format.